src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -122,12 +117,11 @@ export const WorkspaceMenu = ({ coreStart, registeredUseCases$ }: Props) => {
           {currentWorkspace ? (
             <>
               <EuiFlexItem grow={false}>
-                <EuiAvatar
-                  size="m"
-                  type="space"
-                  name={currentWorkspaceName}
-                  color={getValidWorkspaceColor(currentWorkspace?.color)}
-                  initialsLength={2}
+                <EuiIcon
+                  size="xl"
+                  color="success"
+                  data-test-subj={`current-workspace-icon-${getUseCase(currentWorkspace)?.id}`}
+                  type={`ws${getUseCase(currentWorkspace)?.title}`}
```

Comment:
```suggestion
                  type={`ws${getUseCase(currentWorkspace)?.id}`}
```
should be id? and I think we should give it a fallback value.